package.json

```diff
@@ -7,6 +7,7 @@
     "develop": "gatsby develop",
     "dev": "gatsby develop",
     "clean": "gatsby clean",
+    "doc": "LIMIT_BLOG_PAGES=1 gatsby develop",
     "author": "LIMIT_BLOG_PAGES=1 SKIP_DOCS=true gatsby develop",
```

Comment:
Thanks. I think this was done earlier to improve the performance of `yarn develop`. TBH it's very fast now (for me at least) but sure, for consistency we can add it.